bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -1101,6 +1105,101 @@ public ServerConfiguration setMajorCompactionThreshold(double threshold) {
         return this;
     }
 
+    /**
+     * Get threshold of median major compaction.
+     *
+     * <p>same with the @see #getMajorCompactionThreshold()
+     * the high threshold is used when the system is on a low load status
```

Comment:
The change has nothing to do with adjusting compaction parameters based on actual system/bookie load as one can think after reading this, it adjusts them on schedule (which supposed to reflect one's best guess for the system load). I'd adjust the doc text to reflect this.